corehq/apps/app_manager/tests/app_factory.py

```diff
@@ -83,11 +83,12 @@ def form_updates_case(form, case_type=None, parent_case_type=None, update=None):
         else:
             case_type = case_type or form.get_module().case_type
             index = len([load for load in form.actions.load_update_cases if load.case_type == case_type])
-            action = LoadUpdateAction(
-                case_type=case_type,
-                case_tag='load_{}_{}'.format(case_type, index),
-                case_properties=update,
-            )
+            kwargs = {'case_type': case_type, 'case_tag': 'load_{}_{}'.format(case_type, index)}
+            if update:
+                kwargs['case_properties'] = update
+            if preload:
+                kwargs['preload'] = preload
+            action = LoadUpdateAction(**kwargs)
```

Comment:
Agreed, and I like how much code is eliminated by moving preloading to the update method.

How do you feel about keeping a one-line `form_preloads_case` method so that `AppFactory` users don't have to preload using a function called "update"? I know it's the same action in advanced modules but it's different actions in basic modules, and it might feel more logical.